includes/wikia/services/UserService.class.php

```diff
@@ -175,10 +175,24 @@ private function getUsersFromCacheByName( &$names ) {
 
 	private function getUserFromMemCacheById( $id ) {
 		$cacheIdKey = wfSharedMemcKey( "UserCache:".$id );
-		if ( ( $value = F::app()->wg->memc->get( $cacheIdKey ) ) !== false ) {
-			//cache locally
-			$this->cacheLocalUser( $value );
-			return $value;
+		$value = F::app()->wg->memc->get( $cacheIdKey );
```

Comment:
Heh, I wanted to suggest to rename `$value` to `$user` but taking into account what we're trying to fix here it should rather be `$probablyUser` :japanese_ogre: Let's leave it like it it, I guess :lollipop: